rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java

```diff
@@ -160,22 +160,12 @@ private static String addEscape(String colName, String dbType, Escape escape) {
 
         KeywordChecker keywordChecker = KeywordCheckerFactory.getKeywordChecker(dbType);
         if (keywordChecker != null) {
-            boolean check = keywordChecker.check(colName);
+            boolean check = keywordChecker.checkNative(colName);
             if (!check) {
-                boolean uppercase = isUppercase(colName);
-                if (uppercase && isOracle(dbType)) {
-                    // oracle
-                    // we are recommend table name and column name must uppercase.
-                    // if exists full uppercase, the table name or column name does't bundle escape symbol.
-                    return colName;
-                }
-                boolean containsUppercase = containsUppercase(colName);
-                if (!containsUppercase && isPostgresql(dbType)) {
-                    // postgresql
-                    // we are recommend table name and column name must lowercase.
-                    // if exists uppercase character or full uppercase, the table name or column name must bundle escape symbol.
-                    return colName;
-                }
+                // postgresql
+                // we are recommend table name and column name must lowercase.
+                // if exists uppercase character or full uppercase, the table name or column name must bundle escape symbol.
```

Comment:
move these 3 lines to pgsql keywordcheck

Reply:
There is no KeywordChecker for PGSQL, waiting for PGSQL to merge and process.